### ISSUE TITLE
Decimal comma

### ISSUE THREAD

**Issue by hgeorgsch** [KQMATH/moodle-qtype_stack#6]
STACK does not accept decimal comma for floating point numbers, only the decimal point. This is uncontroversial as long as the code written is maxima code.
Introducing the WYSIWYG interface, the student should be writing in the same language and notation as the question is phrased in, and the decimal comma is used in most languages.
Ideally, decimal comma should be used in both input and output depending on the student's language setting. For most purposes, however, it would suffice to let tex2max translate the decimal comma into a decimal point.

**Comment by andstor** [comment]
I have created an issue in tex2max: KQMATH/tex2max#50.

**Comment by andstor** [comment]
The logic in TeX2Max has now been resolved. I will shortly make a new TeX2Max release and then update Stack.

**Comment by hgeorgsch** [comment]
Is this completed?
Does it, BTW, depend on language settings?

**Comment by andstor** [comment]
No, the release is still pending, and the logic needs to be implemented in STACK 😅

In the new version, TeX2Max allows both point and comma signs (although, it's configurable). The output is transpiled to points, as Maxima only supports point symbols. It should therefore be straight forward to just update the TeX2Max scripts in STACK.

By supporting both types, I don't think it's necessary to depend on language settings... Or?

**Comment by hgeorgsch** [comment]
Probably not.  As long as nobody comes up with problems where comma is used for something else, we should be safe.

For consistency, it would be an advantage to only accept comma for language which really use comma, as that is what is done in the calculated qtype.  This is low priority though.
